### PR TITLE
Report tweaks

### PR DESCRIPTION
**Pull request by Mateo9569**

### CODE REVIEW

File: 0400-results.Rmd [hunk 1]
```diff
@@ -6,6 +6,20 @@ A map showing sample sites for fish sampling, fish habitat assessments and disch
 knitr::include_graphics("fig/elk_fish_passage_2022.png")
 ```
 
+**Habitat Model Parameters**
```
Comment (NewGraphEnvironment) [inline 0400-results.Rmd:9]: We have scripts that do this already.... Is this coming from the map screenshot?  I was thinking that you would insert that jpg onto the map like simon does.....

**Review by NewGraphEnvironment — APPROVED** [review]
tx

File: 0400-results.Rmd (hunk 1 above)
Comment (Mateo9569) [reply 0400-results.Rmd:9]: It was from the screenshot yes. You can take that text out then. I tried putting the screenshot in the legend but the text was too small and not legible and it looked bad to be honest.

Comment (NewGraphEnvironment) [reply 0400-results.Rmd:9]: I don't like the tone of this reply

Comment (Mateo9569) [reply 0400-results.Rmd:9]: What? That wasn't meant to be an annoyed tone. I was unsure whether or not to put that text in so I was suggesting that you could take it out if we already have it somewhere else. Just being honest.

**Pull request merged**